docs/userguide/containers/dockervolumes.md

```diff
@@ -11,45 +11,34 @@ parent = "engine_learn"
 
 # Manage data in containers
 
-So far we've been introduced to some [basic Docker concepts](../containers/usingdocker.md),
-seen how to work with [Docker images](../containers/dockerimages.md) as well as learned about
-[networking and links between containers](../networking/default_network/dockerlinks.md). In this section we're
-going to discuss how you can manage data inside and between your Docker
-containers.
+So far you've been introduced to some [basic Docker
+concepts](../containers/usingdocker.md), seen how to work with [Docker
+images](../containers/dockerimages.md) as well as learned about [networking and
+links between containers](../networking/default_network/dockerlinks.md). In this
+section you're going to learn how you can manage data inside and between your
+Docker containers.
 
-We're going to look at the two primary ways you can manage data in
-Docker.
+You're going to look at the two primary ways you can manage data with
+Docker Engine.
 
-* Data volumes, and
-* Data volume containers.
+* Data volumes
+* Data volume containers
 
 ## Data volumes
 
 A *data volume* is a specially-designated directory within one or more
-containers that bypasses the [*Union File System*](../../reference/glossary.md#union-file-system). Data volumes provide several useful features for persistent or shared data:
-
-- Volumes are initialized when a container is created. If the container's
-  base image contains data at the specified mount point, that existing data is
-  copied into the new volume upon volume initialization. (Note that this does
-  not apply when [mounting a host directory](#mount-a-host-directory-as-a-data-volume).)
-- Data volumes can be shared and reused among containers.
-- Changes to a data volume are made directly.
-- Changes to a data volume will not be included when you update an image.
-- Data volumes persist even if the container itself is deleted.
-
-Data volumes are designed to persist data, independent of the container's life
-cycle. Docker therefore *never* automatically deletes volumes when you remove
-a container, nor will it "garbage collect" volumes that are no longer
-referenced by a container.
```

Comment:
Do we really want to remove this section? I think all this information is still relevant?